builtin/intrinsics.mbt

```diff
@@ -184,7 +184,7 @@ pub fn Double::convert_i64_u(val : Int64) -> Double = "%i64_to_f64_u"
 
 pub fn Char::to_int(self : Char) -> Int = "%char_to_int"
 
-pub fn Char::from_int(val : Int) -> Char = "%char_from_int"
+pub fn Char::from_int_unchecked(val : Int) -> Char = "%char_from_int"
 
```

Comment:
this unsafe API does not be exposed, it can be pulled in on demand, so that it is only local to its used package

Reply:
In my current implementation, it is used by `coverage` and `json5`